### ISSUE TITLE
For with kotlin

### ISSUE THREAD

**Issue by abeq** [domaframework/doma#129]
Is there way to fix problem (http://backpaper0.github.io/ghosts/kotlin-doma.html#/29) with Kept/Kotlin on Doma side?

**Comment by nakamura-to** [comment]
You may use `org.seasar.doma.ParameterName`.

```java
@Select
Dept selectById(@ParameterName("departmentId") Integer departmentId);
```

**Comment by nakamura-to** [comment]
`@ParameterName` doesn't work properly for Entity's constructors now.
 It will be addressed in new version(2.8.1 or 2.9.0).

**Comment by nakamura-to** [comment]
Try Doma 2.9.0.
This sample project may be helpful for you.
https://github.com/domaframework/kotlin-sample
